packages/api/src/schemas/envConfig.json

```diff
@@ -33,7 +33,41 @@
         },
         "values": {
           "type": "object",
-          "description": "Configs to customize the Teams app manifest."
+          "description": "Configs to customize the Teams app manifest.",
+          "properties": {
+            "appName": {
+              "type": "object",
+              "description": "Teams app name.",
+              "properties": {
+                "short": {
+                  "type": "string",
+                  "description": "A short display name for teams app.",
+                  "maxLength": 30
+                },
+                "full": {
+                  "type": "string",
+                  "description": "The full name for teams app.",
+                  "maxLength": 100
+                }
+              }
+            },
+            "description": {
```

Comment:
If we really want to expose the `description` config by default, App Studio plugin needs to also provide the initial value for this property in the config file at scaffold stage (same as the `appName` property).